common/services/allocation.js

```diff
@@ -83,8 +83,17 @@ const allocationService = {
   },
   transform({ includeCancelled = false } = {}) {
     return function transformAllocation(result) {
+      const { total: totalSlots, filled: filledSlots } =
+        result.meta?.moves || {}
+
+      result = { ...result }
+      delete result.meta
+
       return {
         ...result,
+        totalSlots,
+        filledSlots,
+        unfilledSlots: totalSlots ? totalSlots - filledSlots : undefined,
         moves: result.moves
           .filter(
             move => includeCancelled || !['cancelled'].includes(move.status)
```

Comment:
We may also be able to remove this as I don't think we need to do any logic with `allocations.moves`.